services/s3/src/main/java/software/amazon/awssdk/services/s3/internal/crt/DefaultS3CrtAsyncClient.java

```diff
@@ -106,11 +109,16 @@ private static S3CrtAsyncHttpClient.Builder initializeS3CrtAsyncHttpClient(Defau
                                        .endpointOverride(builder.endpointOverride)
                                        .credentialsProvider(builder.credentialsProvider)
                                        .readBufferSizeInBytes(builder.readBufferSizeInBytes)
-                                       .httpConfiguration(builder.httpConfiguration)
-                                       .build();
+                                       .httpConfiguration(builder.httpConfiguration);
 
+        if (builder.retryConfiguration != null) {
+            nativeClientBuilder.standardRetryOptions(
+                new StandardRetryOptions()
+                    .withBackoffRetryOptions(new ExponentialBackoffRetryOptions()
```

Comment:
Does this jitter the retries?

Reply:
The interval between the retries are Jittered https://awslabs.github.io/aws-crt-java/software/amazon/awssdk/crt/io/ExponentialBackoffRetryOptions.JitterMode.html